modules/apps/translation/translation-translator-deepl/src/main/java/com/liferay/translation/translator/deepl/internal/model/Translation.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.translator.deepl.internal.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * @author Yasuyuki Takeo
+ */
+public class Translation {
+
+	public Translation(String detectedSourceLanguage, String text) {
+		_detectedSourceLanguage = detectedSourceLanguage;
+		_text = text;
+	}
+
+	@JsonProperty("detected_source_language")
```

Comment:
Now that you're mapping things manually, shouldn't you remove these annotations?